aiohttp/web_runner.py

```diff
@@ -135,6 +136,9 @@ def name(self) -> str:
         self._server = await loop.create_unix_server(
             server, self._path,
             ssl=self._ssl_context, backlog=self._backlog)
+        # create_unix_server sets file permission as rwxr-xr-x, which is not
+        # helpful in practice, so we fix it.
+        os.chmod(self._path, 0o664)
```

Comment:
Based on the above, I still want it to be configurable. Also, the default should give write privileges only to the owner.